src/sleep/battery.c

```diff
@@ -117,17 +125,16 @@ static int siphash24_compress_id128(int (*getter)(sd_id128_t*), const char *name
 
         r = getter(&id);
         if (r < 0)
-                return log_debug_errno(r, "Failed to get %s ID: %m", name);
+                return log_debug_errno(r, "Failed to get %s ID: %m", strna(name));
```

Comment:
It seems this function is always called with `name` set. Adding `assert(name)` would be nicer.